tensorrt_llm/_torch/pyexecutor/kv_cache_manager_v2.py

```
@@ -2830,6 +2834,50 @@ def _get_batch_cache_indices_by_pool_id(
+        if layer_idx is None:
+            pool_id = 0
+        else:
+            pool_id = self.layer_to_pool_mapping_dict[self.layer_offsets[layer_idx]]
+
+        scale = self._index_scale_ints[pool_id]
+        div_factor = self.kv_factor
+
+        out_tensor = torch.empty(sum(num_blocks), dtype=torch.int32, pin_memory=prefer_pinned())
+        out = out_tensor.numpy()
+        offset = 0
+        for req_id, n in zip(request_ids, num_blocks):
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)